dev-perl/xml_sax/additional-files/uninstall_xml_sax_pureperl.pl

```diff
@@ -0,0 +1,5 @@
+#!perl
+
+use XML::SAX;
+
+XML::SAX->remove_parser(q(XML::SAX::PurePerl))->save_parsers();
```

Comment:
Should we maybe delete the ParserDetails.ini file completely instead of just removing xml_sax's entry? This would discard any manual changes a user may have made (because they installed another perl SAX parser manually), but I guess they would be broken anyway if the main XML::SAX module is uninstalled.